hoomd/mpcd/CommunicatorGPU.cc

```diff
@@ -358,7 +358,7 @@ void mpcd::CommunicatorGPU::setCommFlags(const BoxDim& box)
  */
 void mpcd::detail::export_CommunicatorGPU(py::module& m)
     {
-    py::class_<mpcd::CommunicatorGPU, std::shared_ptr<mpcd::CommunicatorGPU> >(m,"CommunicatorGPU",py::base<Communicator>())
+    py::class_<mpcd::CommunicatorGPU, Communicator, std::shared_ptr<mpcd::CommunicatorGPU> >(m,"CommunicatorGPU")
```

Comment:
I think this should be `mpcd::Communicator` for clarity, although apparently it compiled correctly before anyway.

Reply:
confirmed